main.go

```diff
@@ -43,6 +45,8 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/controller"
 	"sigs.k8s.io/controller-runtime/pkg/healthz"
 	// +kubebuilder:scaffold:imports
+
+	operatorv1alpha1 "sigs.k8s.io/cluster-api-operator/api/v1alpha1"
```

Comment:
duplicate import of `operatorv1 "sigs.k8s.io/cluster-api-operator/api/v1alpha1"`, please remove unnecessary whitespaces too if possible